CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.cc

```diff
@@ -23,9 +23,15 @@
 #include "CondFormats/EcalObjects/interface/EcalTPGPedestals.h"
 #include "CondFormats/DataRecord/interface/EcalTPGPedestalsRcd.h"
 
+//modif-alex-27-july-2015
+#include "CalibCalorimetry/EcalLaserCorrection/interface/EcalLaserDbService.h"
+#include "CalibCalorimetry/EcalLaserCorrection/interface/EcalLaserDbRecord.h"
+
 #include "SimCalorimetry/EcalSimAlgos/interface/EcalSimParameterMap.h"
+#if (CMSSW_VERSION>=340)
 #include "SimCalorimetry/EcalSimAlgos/interface/EBShape.h"
```

Comment:
@depasse, can you clarify why do you need this protection? is this still running in a 3.4.X version somewhere (???)